### DISCUSSION TITLE
[FR] Ability to exclude specific websites from auto-saving function

### DISCUSSION THREAD

**Discussion by InInfinitumEtInTenebris** [gildas-lormeau/SingleFile#1240] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I have too many auto-saved files on sites that are sensitive or just not suitable for autosaving.

**Describe the solution you'd like**
I would like for there to be an option to exclude specific websites from the autosaving function.

**Additional context (optional)**
None

**Comment by gildas-lormeau** [comment, marked as answer]
This is possible by adding an "auto-setting rule". For example, in the screenshot below, all domains are automatically saved except "example.com" (and "file:" URIs, present by default).

<img width="640" alt="image" src="https://github.com/gildas-lormeau/SingleFile/assets/396787/76b5ed30-7c0a-4b48-8f83-ca09f175a3d5">

**Reply by InInfinitumEtInTenebris** [reply]
Hello. I know this is three weeks late, but I would like to apologize for writing an Issue without knowing the details of the extension's capabilities. I thank you for explaining to me the process of excluding specific websites, and it has helped tremendously. 

Sincerely, 
Azz
